### DISCUSSION TITLE
我发现在发消息的时候nonebot好像会停止接收消息，导致我漏掉了那些速度比较快的回应

### DISCUSSION THREAD

**Discussion by zhtoatao** [nonebot/discussions#538] [问答]
我是Lagrange.OneBot配合nonebot使用的，我设置了消息处理器用来和人进行连续的问答，我发现如果对面回应的过快，nonebot的输出台上不会接收到回应，但是lagrange能够接收到，不知道是不是因为我的机器人在发消息的时候停止了接收消息的行为，遇到这种情况我要怎么办

**Comment by yanyongyu** [comment]
首先你先确认是不是没有收到对方的消息，还是收到了但是没处理，如果是回复太快没处理的话那就应该是你的代码执行太慢或者阻塞，比接收到对方发送还慢
